scripts/analysis.py

```diff
@@ -563,12 +570,12 @@ def get_deployment_dict(cur):
 
     Returns
     ------
-    num_dict: dictionary
+    deployment_government: dictionary
```

Comment:
it would be better if all instances of "government" were replaced with "institution"  since governments aren't a cyclus concept (we use regions, institutions, and facilities).